simulator/util/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml

```diff
@@ -50,6 +50,17 @@
   <!-- perception module -->
   <group>
     <push-ros-namespace namespace="perception"/>
+    <!-- object segmentation module -->
+    <group>
+      <push-ros-namespace namespace="obstacle_segmentation"/>
+      <!-- Occupancy Grid -->
+      <include file="$(find-pkg-share laserscan_to_occupancy_grid_map)/launch/laserscan_to_occupancy_grid_map.launch.py">
+        <arg name="input_obstacle_pointcloud" value="true" />
+        <arg name="input_obstacle_and_raw_pointcloud" value="false" />
+        <arg name="input/obstacle_pointcloud" value="/perception/obstacle_segmentation/pointcloud" />
```

Comment:
@satoshi-ota 
namespace is pushed so this is better to avoid typo , I think
```suggestion
        <arg name="input/obstacle_pointcloud" value="pointcloud" />
```

Reply:
but another case like `/perception/object_recognition/objects` uses absolute path so it's up to you